### ISSUE TITLE
Windows 10 support

### ISSUE THREAD

**Issue by michbsd** [Swyx/swyxware-docker#3]
Hi, 

Any chance this will run on a Windows 10 machine? (for testing) 

Thanks,

**Comment by michbsd** [comment]
The command: 
`Install-Package -Name Docker -ProviderName DockerMsftProvider `

fails, because `Get-WindowsFeature` is not available on Windows 10 

Installing the Docker package manually (not from DockerMsftProvider) and removing check from the installation script seems to fix it.

**Comment by martinh2011** [comment]
Yes, should work. The script was never intended to run on Windows 10, however. I'm not sure if there are other issues when using it on a client OS.

In general I'm not really happy with the script. Yes, it makes it easy to get the SwyxWare and SQLExpress container up and running. But it also hides a lot of the docker stuff which an admin should better learn and understand if he plans to use the technology.

**Comment by michbsd** [comment]
Indeed. I ran into some other things. 
Going to set it up manually.
